rrfs-test/scripts/link_mpasjedi_expr.sh

```diff
@@ -44,10 +44,10 @@ cp ${RDASApp}/rrfs-test/ush/mpasjedi_spread.py .
 
 mkdir -p data
 cd data
-mkdir -p bumploc bkg obs ens ref
+mkdir -p bumploc obs ens ref
 ln -snf ${RDASApp}/fix/bumploc/${BUMPLOC} bumploc/${BUMPLOC}
-ln -snf ${RDASApp}/fix/expr_data/${exprname}/bkg/restart.2024-05-27_00.00.00.nc .
-ln -snf ${RDASApp}/fix/expr_data/${exprname}/bkg/restart.2024-05-27_00.00.00.nc static.nc
+ln -snf ${RDASApp}/fix/expr_data/${exprname}/bkg/mpasout.2024-05-27_00.00.00.nc .
+ln -snf ${RDASApp}/fix/expr_data/${exprname}/static.nc static.nc
```

Comment:
could those static files in the source location be added  qualifiers of  grid-related labels?